modules/security-container-content-inside.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-container-content.adoc
+
+[id="security-container-content-inside_{context}"]
+= Security inside the container
+
+Applications and infrastructures are composed of readily available components,
+many of which are open source packages such as, the Linux operating system,
+JBoss Web Server, PostgreSQL, and Node.js.
+
+Containerized versions of these packages are also available. However, you need
+to know where the packages originally came from, what versions are used, who built them, and whether
+there is any malicious code inside them.
+
+Some questions to answer include:
+
+- Will what is inside the containers compromise your infrastructure?
+- Are there known vulnerabilities in the application layer?
+- Are the runtime and OS layers current?
```

Comment:
I'd write out OS. A few other places in the PR

Reply:
There was one place where it defined "operating system (OS)" then reused OS within a couple of paragraphs. I left that, but changes all other instances of OS to operating system.